exasol/ds/sandbox/runtime/ansible/roles/entrypoint/files/entrypoint.py

```diff
@@ -83,8 +99,11 @@ def exit_on_error(rc):
         "--no-browser",
         "--allow-root",
     ]
+
+    env = os.environ.copy()
```

Comment:
Did you check the content of the environment, if it fits to the new user

Reply:
See comment in ticket: 
https://github.com/exasol/ai-lab/issues/241#issuecomment-1980249721